netbox/utilities/templatetags/helpers.py

```diff
@@ -149,6 +149,9 @@ def example_choices(field, arg=3):
             break
         if not value or not label:
             continue
+        # Handling for custom fields
+        if hasattr(label, 'value'):
```

Comment:
This should be handled in the form widget, not the template tag. The widget should be modified to return a set of choices in the standard tuple format.